projects/policekillings/package.json

```diff
@@ -1,14 +1,16 @@
 {
-  "name": "police_killings_dashboard",
+  "name": "project-one",
```

Comment:
Let's keep the project name related to the application; `project-one` is just an example name that can be anything. I'd recommend `policekillings` since it matches the folder name and might help keep things simple.